src/intro_movie_scene_2.c

```diff
@@ -0,0 +1,457 @@
+    Set2dSpriteAnimSeqNo(sceneData->lyraSprite1, 4);
+    sceneData->lyraSprite2 = CreateSprite(&spriteTemplate);
+    Set2dSpriteAnimActiveFlag(sceneData->lyraSprite2, FALSE);
+    Set2dSpriteVisibleFlag(sceneData->lyraSprite2, TRUE);
+    Set2dSpriteAnimSeqNo(sceneData->lyraSprite2, 5);
+}
+
+void IntroMovie_Scene2_StartSpritePanEffect(IntroMovieScene2Data *sceneData, int whichSprites, int xSpeed, int ySpeed, int duration) {
+    IntroMovieScene2SpritePanController *spritePanEffect;
+
+    if (whichSprites == 0) {
+        spritePanEffect = &sceneData->spritePanCnt[0];
+        spritePanEffect->sprite0 = sceneData->ethanSprite0;
+        spritePanEffect->sprite1 = sceneData->ethanSprite1;
+        spritePanEffect->sprite2 = sceneData->ethanSprite2;
+    } else {
+        spritePanEffect = &sceneData->spritePanCnt[1];
+        spritePanEffect->sprite0 = sceneData->lyraSprite0;
+        spritePanEffect->sprite1 = sceneData->lyraSprite1;
+        spritePanEffect->sprite2 = sceneData->lyraSprite2;
+    }
+    GF_ASSERT(!spritePanEffect->active);
+    spritePanEffect->active = TRUE;
+    spritePanEffect->duration = duration;
+    spritePanEffect->counter = 0;
+    spritePanEffect->finished = 0;
+    VecFx32 *mtx = Sprite_GetMatrixPtr(spritePanEffect->sprite0);
+    spritePanEffect->xOffset = mtx->x / FX32_ONE;
+    spritePanEffect->yOffset = mtx->y / FX32_ONE;
+    spritePanEffect->xSpeed = xSpeed;
+    spritePanEffect->ySpeed = ySpeed;
+    spritePanEffect->task = SysTask_CreateOnMainQueue(ov60_021E8B7C, spritePanEffect, 0);
+}
+
+BOOL IntroMovie_Scene2_WaitSpritePanEffect(IntroMovieScene2Data *sceneData, int whichSprites) {
+    IntroMovieScene2SpritePanController *spritePanCnt;
+    if (whichSprites == 0) {
+        spritePanCnt = &sceneData->spritePanCnt[0];
+    } else {
+        spritePanCnt = &sceneData->spritePanCnt[1];
+    }
+
+    if (!spritePanCnt->active) {
+        return TRUE;
+    }
+    if (spritePanCnt->finished) {
+        return TRUE;
+    }
+
+    return FALSE;
+}
+
+void ov60_021E8B7C(SysTask *task, void *pVoid) {
+    IntroMovieScene2SpritePanController *spritePanCnt = (IntroMovieScene2SpritePanController *)pVoid;
+    VecFx32 sp4 = {0, 0, 0};
+    ++spritePanCnt->counter;
+    int x = spritePanCnt->xSpeed * spritePanCnt->counter / spritePanCnt->duration;
+    int y = spritePanCnt->ySpeed * spritePanCnt->counter / spritePanCnt->duration;
+    x += spritePanCnt->xOffset;
+    y += spritePanCnt->yOffset;
+    if (spritePanCnt->counter >= spritePanCnt->duration) {
+        SysTask_Destroy(spritePanCnt->task);
+        spritePanCnt->task = NULL;
+        spritePanCnt->finished = TRUE;
+        spritePanCnt->active = FALSE;
+    }
+    sp4.x = x * FX32_ONE;
+    sp4.y = y * FX32_ONE;
+    Sprite_SetMatrix(spritePanCnt->sprite0, &sp4);
+    Sprite_SetMatrix(spritePanCnt->sprite1, &sp4);
+    Sprite_SetMatrix(spritePanCnt->sprite2, &sp4);
+}
+
+HeapID _deadstrip_02(int idx);
+HeapID _deadstrip_02(int idx) {
```

Comment:
deadstrip -> unused for function names etc for the other deadstripped function names

Reply:
wont-fix